app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  belongs_to :user
```

Comment:
What about users? The rent has an user, and the user `has_many :rents`

Reply:
Also has_many requires to add `dependent` statement

Reply:
Done!.